components/brave_rewards/common/pref_names.cc

```diff
@@ -8,6 +8,14 @@
 namespace brave_rewards {
 namespace prefs {
 
+const char kScheduledCaptchaId[] = "brave.rewards.scheduled_captcha.id";
```

Comment:
re: above - these prefs should definitely move into the captcha component (along with registration) or you're going to have to do a pref migration. I really think you'd be much better off just creating a new keyed service now

Reply:
Agreed, done.